src/v3/src/hooks/useOnSubmit.ts

```diff
@@ -197,6 +197,8 @@ export const useOnSubmit = (): (options: OnSubmitHandlerOptions) => Promise<void
       authClient?.transactionManager.clear({ clearIdxResponse: false });
       SessionStorage.removeStateHandle();
       if (isOauth2Enabled(widgetProps)) {
+        // We have to directly delete the recoveryToken since it is set once upon authClient instantiation
+        delete authClient.options.recoveryToken;
```

Comment:
Can you also add this to the transform terminal transaction transformer? Above [this](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/transformer/terminal/transformTerminalTransaction.ts#L122) line.